src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+const list = [...ul.querySelectorAll('li')];
+
+function toNumber(str) {
+  return +str.replace(/['$']/g, '').replace(/[',']/g, '.');
```

Comment:
The `toNumber` function uses `replace` to handle currency formatting, but it might not correctly handle all cases. For example, it assumes that the currency symbol is always at the start and that commas are used as thousand separators. Consider using a more robust method for parsing currency values.